tests/cephfs/clients/mds_scrub_after_failover.py

```diff
@@ -0,0 +1,137 @@
+import json
+import random
+import string
+import time
+import traceback
+
+from ceph.parallel import parallel
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from tests.io.fs_io import fs_io
+from utility.log import Log
+
+log = Log(__name__)
+
+"""
+Test steps for 83573489
+1. Fill up the cluster with 60% data
+2. Prepare the clients with different mounting options
+3. Set max mds to 2
+4. Set standby count to 2
+5. Start IO on all the clients
+6. Kill one of the active MDS
+7. Start scrubbing
+8. check if it shows any issue while scrubbing
+"""
+
+
+def run(ceph_cluster, **kw):
+    try:
+        fs_util = FsUtils(ceph_cluster)
+
+        config = kw.get("config")
+
+        clients = ceph_cluster.get_ceph_objects("client")
+
+        build = config.get("build", config.get("rhbuild"))
+
+        fs_util.prepare_clients(clients, build)
+
+        fs_util.auth_list(clients)
+
+        log.info("checking Pre-requisites")
+
+        client1 = clients[0]
+
+        cephfs = {
+            "fill_data": 60,
+            "io_tool": "smallfile",
+            "mount": "fuse",
+            "filesystem": "cephfs",
+            "mount_dir": "/mnt/mycephfs1",
+        }
+        # fill up to 60% of the cluster
+        fs_io(client=clients[0], fs_config=cephfs, fs_util=fs_util)
+        # Prepare the clients with different mounting options
+        # set max mds 2
+        client1.exec_command(sudo=True, cmd="ceph fs set cephfs max_mds 2")
+        # # set standby count to 2
+        client1.exec_command(sudo=True, cmd="ceph fs set cephfs standby_count_wanted 3")
```

Comment:
Can you revisit the "standby_count_wanted 3" based on the existing mds configs?
I assume, we already created 2 Filesystems created as part of cluster deployment.
cephfs - with 2 active-1 standby node and 
cephfs-ec - with 1 active - 1 standby.

If you need to use this standby_count_wanted - we should make sure the mds's are free and available.

Maybe you can delete the existing FileSystems before the start of your test and recreate it with the same number of nodes present earlier.

Reply:
revisited. created file system again.

If if has a file system already, it will delete it and will create it with 4 node placement